### PR TITLE
examples/volano: split benchmark from campaign

### PR DESCRIPTION
**Pull request by apaolillo**

### CODE REVIEW

File: examples/volano/kit/volano.py [hunk 1]
```diff
@@ -1,26 +1,22 @@
...
-from benchkit.commandattachments.tcpdump import TcpDump
 from benchkit.commandwrappers import CommandWrapper
 from benchkit.dependencies.packages import PackageDependency
 from benchkit.platforms import Platform
 from benchkit.sharedlibs import SharedLib
 from benchkit.utils.types import PathType
+from benchkit.utils.dir import caller_dir
 
 
 class CounterBenchmark(Benchmark):
```
Comment (matsvanmolle) [inline examples/volano/kit/volano.py]: Could you change the class name to something more meaningful?

Comment (apaolillo) [reply examples/volano/kit/volano.py]: suggestion?

Comment (matsvanmolle) [reply examples/volano/kit/volano.py]: VolanoBenchmark, This was an error on my side way back.

Comment (apaolillo) [reply examples/volano/kit/volano.py]: done

**Review by matsvanmolle — APPROVED** [review]

**Pull request merged**